snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/taxonomy/ReasonerTaxonomyBuilder.java

```diff
@@ -752,6 +752,8 @@ public ReasonerTaxonomyBuilder addActiveAxioms(final RevisionSearcher searcher)
 							}
 						} else if ("subdatapropertyof".equals(firstToken)) {
 							isSubPropertyOf = true;
+						} else if ("subannotationpropertyof".equals(firstToken)) {
```

Comment:
Would it make sense to make this configurable in the snowowl.yml config file?

Reply:
Hopefully there will be no other property types!